nginx/default.conf.template

```diff
@@ -0,0 +1,67 @@
+server {
+  listen       80;
+  listen  [::]:80;
+  server_name  localhost;
+
+  root /usr/share/nginx/html;
+  index index.html;
+
+  error_page 500 502 503 504 /50x.html;
+
+  gzip on;
+  gzip_disable "msie6";
+  gzip_vary on;
+  gzip_min_length 1000;
+  gzip_types text/plain text/html text/css application/javascript application/json;
+
+
+  # SPA routing, everything falls back to the root index.html
+  location / {
+    try_files $uri $uri/ /index.html =404;
+
+    add_header Cache-Control "no-store";
+    add_header X-Frame-Options "DENY" always;
+    add_header X-Content-Type-Options "nosniff" always;
+    add_header X-XSS-Protection "1; mode=block" always;
+    add_header Referrer-Policy "strict-origin-when-cross-origin" always;
+    add_header Strict-Transport-Security "max-age=63072000; includeSubDomains; preload" always;
+
+    add_header Content-Security-Policy "
+        default-src 'self';
+        script-src
+            'self'
+            'unsafe-eval'
+            blob:
+            https://cdn.jsdelivr.net/pyodide/v0.26.2/
+            https://cdn.jsdelivr.net/npm/@stlite/mountable@0.72.1/;
+        style-src
+            'self'
+            'unsafe-inline'
```

Comment:
we can add hash computed from the css during docker build, but I think it's overkill for iframe that can execute arbitrary code anyway